lib/cloudant.js

```diff
@@ -75,7 +75,42 @@ Cloudant.prototype.getTypes = function() {
  */
 Cloudant.prototype.connect = function(cb) {
   debug('Cloudant.prototype.connect');
-  cb(null, this.cloudant);
+  var self = this;
+  // check if cloudant server is up
+  if (self.options.url) {
+    request.get(self.options.url, function(err, res, body) {
```

Comment:
Can we ensure that both the error and the appropriate HTTP status code are returned as part of the error message?